docs/usage/opentelemetry.md

```diff
@@ -9,24 +9,34 @@ description: How to use OpenTelemetry with Renovate
 !!! warning "This feature is flagged as experimental"
     Experimental features might be changed or even removed at any time.
 
-Renovate supports OpenTelemetry which is an emerging monitoring standard.
+Renovate partially supports OpenTelemetry, the emerging monitoring standard.
 
-OpenTelemetry supports three types of observability data:
+OpenTelemetry has three types of observability data:
 
 - traces
 - metrics
 - logs
 
-Renovate can only send traces and only via the OpenTelemetryProtocol (OTLP), other observability data or transfer protocols are not supported.
+## Limitations
+
+Renovate only:
+
+- sends traces
+- uses the OpenTelemetryProtocol (OTLP)
+
+Renovate does not support other observability data or transfer protocols.
```

Comment:
Hope you don't mind if I chime in here but I think a small misunderstanding has sneaked in:

- While OTLP support traces, metrics, and logs, renovate currently only supports traces. 
- _Uses the OpenTelemetry Protocol_ (mind the space) isn't a limitation. I think the original sentence wanted to say that besides OTLP no other specifications for telemetry data ingestion are supported.
- _Transfer protocols_ is misleading. OTLP can send traces via HTTP (in JSON-encoded protobuf or binary-encoded protobuf format) or via gRPC. renovate uses [@opentelemetry/exporter-trace-otlp-http](https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-http) under the hood. This means, renovate sends traces via [OTLP/HTTP](https://opentelemetry.io/docs/reference/specification/protocol/otlp/#otlphttp) in JSON-encoded protobuf format only. 

_Other observability data_ could be exemplified with stats on caching, error events, no. of found dep updates, etc.

Reply:
```suggestion
## Limitations

While OTLP support traces, metrics, and logs, Renovate only supports traces.
This means Renovate does not support other observability data like: stats on caching, error events, number of found updates, and so on.

Besides OTLP no other specifications for telemetry data ingestion are supported.

Renovate uses [`@opentelemetry/exporter-trace-otlp-http`](https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-http) under the hood.
This means that Renovate sends traces via [OTLP/HTTP](https://opentelemetry.io/docs/reference/specification/protocol/otlp/#otlphttp) in JSON-encoded protobuf format only.
```